psi4/src/psi4/fisapt/fisapt.cc

```diff
@@ -194,9 +194,8 @@ void FISAPT::partition() {
         double delta = options_.get_double("FISAPT_CHARGE_COMPLETENESS");
         outfile->Printf("    Charge Completeness = %5.3f\n\n", delta);
         for (int a = 0; a < na; a++) {
-            if (QFp[0][a] > delta) {
-            } else if (QFp[1][a] > delta) {
-            } else if (QFp[2][a] > delta) {
+            if (QFp[0][a] > delta || QFp[1][a] > delta || QFp[2][a] > delta) {
+                continue;
```

Comment:
I think you have to be careful using `continue;` like the python `pass`. It works here b/c end of `for` loop is right after end of `if` sequence but if there had been anything in between, it would have been unintentionally skipped.